scripts/west-commands.yml

```diff
@@ -16,3 +16,8 @@ west-commands:
       - name: ncs-sbom
         class: NcsSbom
         help: generate Software Bill Of Materials including license information
+  - file: scripts/west_commands/bluetooth/mesh/dfu_metadata.py
```

Comment:
Hmm, I was actually thinking about something like this:
https://github.com/nrfconnect/sdk-nrf/blob/7d209a57228062b2453d8ba25b172e8d09a26d49/cmake/partition_manager.cmake#L708-L715
And then user calls `west build -t ble_mesh_metadata_report` or something. Script also knows about the path to the json and as I understand won't work if firmware is not build.

Reply:
And perhaps the same script which generates metadata can be used.

Reply:
Ok, I'm on it 👍